.github/workflows/run-validations.yml

```diff
@@ -0,0 +1,242 @@
+name: Run validations
+
+on:
+  workflow_call:
+    inputs:
+      repo:
+        required: true
+        type: string
+
+      agent-reqs:
+        required: false
+        default: false
+        type: boolean
+      ci:
+        required: false
+        default: false
+        type: boolean
+      codeowners:
+        required: false
+        default: false
+        type: boolean
+      config:
+        required: false
+        default: false
+        type: boolean
+      dashboards:
+        required: false
+        default: false
+        type: boolean
+      dep:
+        required: false
+        default: false
+        type: boolean
+      eula:
+        required: false
+        default: false
+        type: boolean
+      http:
+        required: false
+        default: false
+        type: boolean
+      imports:
+        required: false
+        default: false
+        type: boolean
+      integration-style:
+        required: false
+        default: false
+        type: boolean
+      jmx-metrics:
+        required: false
+        default: false
+        type: boolean
+      legacy-signature:
+        required: false
+        default: false
+        type: boolean
+      license-headers:
+        required: false
+        default: false
+        type: boolean
+      licenses:
+        required: false
+        default: false
+        type: boolean
+      metadata:
+        required: false
+        default: false
+        type: boolean
+      models:
+        required: false
+        default: false
+        type: boolean
+      package:
+        required: false
+        default: false
+        type: boolean
+      readmes:
+        required: false
+        default: false
+        type: boolean
+      recommended-monitors:
+        required: false
+        default: false
+        type: boolean
+      saved-views:
+        required: false
+        default: false
+        type: boolean
+      service-checks:
+        required: false
+        default: false
+        type: boolean
+      typos:
+        required: false
+        default: false
+        type: boolean
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  run:
+    name: Validate
+    runs-on: ubuntu-22.04
+
+    env:
+      PYTHON_VERSION: "3.8"
+      TARGET: ${{ github.event_name == 'pull_request' && 'changed' || '' }}
```

Comment:
What does `github.event_name == ''` mean?

Reply:
This is essentially a ternary operator and says that if this is a pull request the target will be the word `changed` which means only relevant integrations will be validated and on master it will be an empty argument which will check everything